### PR TITLE
Contact me section

### PR DESCRIPTION
**Pull request by Luchy-B**
Add contact me form to portfolio
Add validations to form

### CODE REVIEW

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Team

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

🔴 🔴  ***You have submitted a project, that is not passing Linters check*** 🔴 🔴 🔴 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,215 +1,280 @@
...
-        <div class="aboutMeContent">
-            <h2>About <br> Myself</h2>
-            <p>Hello I'm a software developer! I can help <br>
-                you build a product , feature or website <br>
-                Look through some of my work and <br>
-                experience! If you like what you see and <br>
-                have a project you need coded, don't <br>
-                hestiate to contact me.
+  <!-- Contact Form -->
+  <section class="contact-Form">
```
Comment (huseyinbiyik) [inline index.html]: Kindly stick to the design and fix the gray area like on the Figma template. 👨‍🎨
Screenshot 2023-01-18 at 9 10 15 PM
Screenshot 2023-01-18 at 9 10 44 PM

File: index.html [hunk 2]
```diff
@@ -1,215 +1,280 @@
...
+        <div class="contact-Me">
+            <h1>Contact me</h1>
+            <p>
```
Comment (huseyinbiyik) [inline index.html:180]: Please follow the Figma template and fix the padding left and right side of the paragraph. It needs to be 24px as you can see below.
Screenshot 2023-01-18 at 9 04 09 PM
Screenshot 2023-01-18 at 9 05 17 PM

File: index.html [hunk 3]
```diff
@@ -1,215 +1,280 @@
...
+                If you have an application you are interested in developing, a feature
+                that you need built or a project that needs coding. I'd love to help
+                with it
```
Comment (huseyinbiyik) [inline index.html]: Please stick to the Figma template and fix the alignment of the paragraph text with `text-align:center` and font-size and font-weight. 
Screenshot 2023-01-18 at 9 06 16 PM

File: index.html [hunk 4]
```diff
@@ -1,215 +1,280 @@
...
+            </label>
+            <label for="email">
+                <input type="email" id="email" name="email" placeholder="Enter the email" required/>
+            </label>
+            <label for="message">
+                <textarea id="message" cols="38" rows="10" required maxlength="500" minlength="10" name="message"placeholder="Write your message here"></textarea>
+            </label>                 
+                <button type="submit" class="button">Get in touch</button>
+        </form>        
+            <img src="images/end-Line.png" alt="Black-Line" class="end-Line">
```
Comment (huseyinbiyik) [inline index.html]: Form submission works properly but let's make the design flawless. Here, there is an alignment issue. You need 24px padding from left and right for name, email, text area, and button. So they will be aligned.
Screenshot 2023-01-18 at 9 08 24 PM

File: index.html [hunk 5]
```diff
@@ -1,215 +1,280 @@
...
+            <label for="name">
+                <input type="text" id="name" name="name" minlength="3" maxlength="30" placeholder="Full name" required/>
+            </label>
+            <label for="email">
+                <input type="email" id="email" name="email" placeholder="Enter the email" required/>
+            </label>
+            <label for="message">
+                <textarea id="message" cols="38" rows="10" required maxlength="500" minlength="10" name="message"placeholder="Write your message here"></textarea>
+            </label>                 
+                <button type="submit" class="button">Get in touch</button>
```
Comment (huseyinbiyik) [inline index.html]: Kindly add active and pressed states for your submit button as requested on the project requirement. Please check the related area on the Figma as guide.
Screenshot 2023-01-18 at 9 11 37 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Collaborative work 🧠🧠
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -1,215 +1,284 @@
...
+                </div>
+                <form action="https://formspree.io/f/mdojyaar" method="POST" class="input-Form">
+                    <label for="name">
+                        <input type="text" id="name" name="name" minlength="3" maxlength="30" placeholder="Full name" required/>
+                    </label>
+                    <label for="email">
+                        <input type="email" id="email" name="email" placeholder="Enter the email" required/>
+                    </label>
+                    <label for="message">
+                        <textarea id="message" cols="40" rows="10" required maxlength="500" minlength="10" name="message"placeholder="Write your message here"></textarea>
```
Comment (leonardodiasb) [inline index.html]: - [ ] You did a great job fixing the issues the previous code reviewer mentioned. But the `textarea` of your contact form is still larger. I would recommend you remove the `cols` and `rows` attributes and use `width` and `height` to style it on your CSS file. Kindly fix it so the `textarea` is aligned with the rest of the inputs.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Luchy-B, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Collaborative work 🧠🧠
 - Linter set up well and checks pass 🟢
 - Github flow followed 👌
 - .gitignore used to ignore requested files 📁
 - Commit messages are meaningful and written in imperative tense 📝
 - PR description is descriptive 📝

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @Luchy-B  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️professional PR👍 
✔️professional README👍 
✔️No linter Errors 👍 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**